common.sh

```diff
@@ -234,9 +251,8 @@ EOF
 
 centos_update()
```

Comment:
Should we just remove this function?

Reply:
I think It'll have to stick around for now since there's a call to ${label}_update and it's probably not worth adding a conditional there.